apps/skilavottord/web/utils/encodeUtils.ts

```
@@ -0,0 +1,17 @@
+export const encode = (input: string): string => {
+  // Normalize the string to decompose characters with diacritics
+  const normalized = input.normalize('NFD')
+
+  // Replace specific Icelandic characters not handled by diacritic removal
+  let replaced = normalized.replace(/[ðÐ]/g, 'd')
+  replaced = replaced.replace(/[þÞ]/g, 'th')
+  replaced = replaced.replace(/[æÆ]/g, 'ae')
+
+  // Remove diacritical marks
+  const noDiacritics = replaced.replace(/[\u0300-\u036f]/g, '')
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)